### PR TITLE
MNT: Bump glue-jupyter minversion to 0.6.1

### LINKED ISSUE #660: Programmatically set application height so images fill available space
**Reporter: Jesse Averbukh**

 DISCLAIMER: This issue was autocreated by the Jdaviz Issue Creation Bot on behalf of the reporter. If any information is incorrect, please contact [Duy Nguyen](https://github.com/duytnguyendtn)

### LINKED ISSUE #677: MNT: Unpin glue-jupyter maxversion 0.5 when region handling is fixed upstream
This is a follow-up of #675 . Once fixed, we can revisit a patch similar to #664 .

https://github.com/spacetelescope/jdaviz/blob/e0efb4f2f7703b869437bd03feb9701b85a93a0b/setup.cfg#L26

### PR DESCRIPTION
**Pull request by pllim**
Fix #660 

Fix #677 

This is #664 with upstream fix.

### CODE REVIEW

**Review by javerbukh — APPROVED** [review]
Works for me!

**Review by rosteen — APPROVED** [review]
Confirmed that I can make regions in image viewers again, approving.

File: jdaviz/configs/imviz/plugins/tools.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def activate(self):
 @viewer_tool
 class BqplotMatchWCS(BqplotPanZoomMode):
 
-    icon = os.path.join(ICON_DIR, 'pan_wcs.png')
+    icon = os.path.join(ICON_DIR, 'pan_wcs.svg')
```
Comment (astrofrog) [inline jdaviz/configs/imviz/plugins/tools.py:33]: You can remove the PNG file from the repo too

Comment (pllim) [reply jdaviz/configs/imviz/plugins/tools.py:33]: See #679

**Pull request merged**